conflictlog/pool.go

```diff
@@ -216,7 +235,13 @@ func (pool *connPool) Get(bucketName string) (conn io.Closer, err error) {
 
 // Put releases the connection back to the pool for reuse. It is caller's job
 // to ensure that right bucket name is passed here.
-func (pool *connPool) Put(bucketName string, conn io.Closer) {
+func (pool *connPool) Put(bucketName string, conn io.Closer, err error) {
+	if err != nil { // this implies damaged connection
+		// NOT IMPLEMENTED YET. As of now we ignore it and not put the damaged connection
```

Comment:
Is an MB tagged already for this? If not let's create one so it doesn't get lost

Reply:
Yeah this will be handled when connection limit is implemented